app/models/validations/household_validations.rb

```diff
@@ -6,6 +6,7 @@ module Validations::HouseholdValidations
   def validate_reasonable_preference(record)
     if record.homeless == "No" && record.reasonpref == "Yes"
       record.errors.add :reasonpref, I18n.t("validations.household.reasonpref.not_homeless")
+      record.errors.add :homeless, I18n.t("validations.household.reasonpref.not_homeless")
```

Comment:
It this a duplicate?

Reply:
Yes, same error message but added on a different field (since those fields appear on different pages we want the error linked no matter which page you've triggered the error from).